README.md

```diff
@@ -28,14 +28,13 @@ Galactic | Citadel | 9.x | [galactic](https://github.com/ros/ros_ign/tree/galact
 Galactic | Edifice | 11.x | [galactic](https://github.com/ros/ros_ign/tree/galactic) | only from source
 Galactic | Fortress | 12.x | [galactic](https://github.com/ros/ros_ign/tree/galactic) | only from source
 Humble | Fortress | 12.x | [humble](https://github.com/ros/sdformat_urdf/tree/humble) | https://packages.ros.org
-Humble | Garden | 13.x | [humble](https://github.com/ros/sdformat_urdf/tree/humble) | only from source
-Humble | Harmonic | 14.x | [humble](https://github.com/ros/sdformat_urdf/tree/humble) | only from source
-Iron | Fortress | 12.x | [iron](https://github.com/ros/sdformat_urdf/tree/iron) | https://packages.ros.org
+Humble | Garden | 13.x | [iron](https://github.com/ros/sdformat_urdf/tree/iron) | only from source
+Humble | Harmonic | 14.x | [rolling](https://github.com/ros/sdformat_urdf/tree/rolling) | only from source
+Iron | Fortress | 12.x | [humble](https://github.com/ros/sdformat_urdf/tree/humble) | https://packages.ros.org
 Iron | Garden | 13.x | [iron](https://github.com/ros/sdformat_urdf/tree/iron) | only from source
-Iron | Harmonic | 14.x | [iron](https://github.com/ros/ros_ign/tree/ros2) | only from source
-Jazzy | Fortress | 12.x | [jazzy](https://github.com/ros/sdformat_urdf/tree/jazzy) | https://packages.ros.org
-Jazzy | Garden | 13.x | [jazzy](https://github.com/ros/sdformat_urdf/tree/jazzy) | only from source
-Jazzy | Harmonic | 14.x | [jazzy](https://github.com/ros/sdformat_urdf/tree/jazzy) | only from source
-Rolling | Fortress | 12.x | [rolling](https://github.com/ros/sdformat_urdf/tree/rolling) | https://packages.ros.org
-Rolling | Garden | 13.x | [rolling](https://github.com/ros/sdformat_urdf/tree/rolling) | only from source
-Rolling | Harmonic | 14.x | [rolling](https://github.com/ros/sdformat_urdf/tree/rolling) | only from source
+Iron | Harmonic | 14.x | [rolling](https://github.com/ros/sdformat_urdf/tree/rolling) | only from source
```

Comment:
I'm not sure I follow the changes here. I was under the impression that each ROS release had a branch, but I see the `rolling` branch specified here for Iron.